src/Symfony/Component/Config/Resource/FileResource.php

```diff
@@ -60,7 +60,9 @@ public function getResource()
      */
     public function isFresh($timestamp)
     {
-        return file_exists($this->resource) && @filemtime($this->resource) <= $timestamp;
+        $filemtime = @filemtime($this->resource);
+
+        return $filemtime && $filemtime <= $timestamp;
```

Comment:
These changes can probably be fixed in a lower branch.

Reply:
You can probably inline the definition as well, i.e. 
```php
return false !== ($filemtime = @filemtime($this->resource))&& $filemtime <= $timestamp;
```